sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/ContainerRegistryAudience.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Containers.ContainerRegistry
+{
+    /// <summary> Cloud audiences available for ACR. </summary>
+    public readonly partial struct ContainerRegistryAudience : IEquatable<ContainerRegistryAudience>
+    {
+        private readonly string _value;
+
+        /// <summary> Determines if two <see cref="ContainerRegistryAudience"/> values are the same. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public ContainerRegistryAudience(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string AzureChinaValue = "https://management.chinacloudapi.cn";
+        private const string AzureGermanyValue = "https://management.microsoftazure.de";
+        private const string AzureGovernmentValue = "https://management.usgovcloudapi.net";
+        private const string AzurePublicCloudValue = "https://management.azure.com";
+
+        /// <summary> Azure China. </summary>
+        public static ContainerRegistryAudience AzureChina { get; } = new ContainerRegistryAudience(AzureChinaValue);
```

Comment:
Are the values of this going to change once they add ACR specific scopes? If not should we name these something like `ArmChina` or something that denotes that these are the ARM audience so we don't steal the good name from what we eventually want users to choose (the ACR specific scope)

Reply:
Updated to be `AzureResourceManagerChina`.  Let me know if this doesn't sound good to you!